tests/fixtures/blueprints/test_aws_lambda_Function.json

```diff
@@ -124,69 +124,66 @@
                                             }
                                         ]
                                     ]
-                                },
+                                }, 
                                 {
                                     "Fn::Join": [
-                                        "",
+                                        "", 
                                         [
-                                            "arn:aws:logs:",
+                                            "arn:aws:logs:", 
                                             {
                                                 "Ref": "AWS::Region"
-                                            },
-                                            ":",
+                                            }, 
+                                            ":", 
                                             {
                                                 "Ref": "AWS::AccountId"
-                                            },
-                                            ":log-group:",
+                                            }, 
+                                            ":log-group:", 
                                             {
                                                 "Fn::Join": [
-                                                    "/",
+                                                    "/", 
                                                     [
-                                                        "/aws/lambda",
+                                                        "/aws/lambda", 
                                                         {
                                                             "Ref": "Function"
                                                         }
                                                     ]
                                                 ]
-                                            },
+                                            }, 
                                             ":*"
                                         ]
                                     ]
                                 }
                             ]
                         }
                     ]
-                },
-                "PolicyName": "test-test_aws_lambda_Function-policy",
+                }, 
```

Comment:
Do you know what the deal is with the whitespace changes?

Reply:
I have no idea but it was generated using `make test`